core/src/main/java/greencity/controller/HabitAssignController.java

```diff
@@ -359,27 +359,27 @@ public ResponseEntity<HabitAssignManagementDto> updateAssignByHabitId(
     /**
      * Method to enroll {@link HabitAssignVO} for current date.
      *
-     * @param habitId - id of {@link HabitVO}.
-     * @param userVO  {@link UserVO} user.
-     * @param date    - {@link LocalDate} we want to enroll.
-     * @param locale  - needed language code.
+     * @param habitAssignId - id of {@link HabitAssignVO}.
+     * @param userVO        {@link UserVO} user.
+     * @param date          - {@link LocalDate} we want to enroll.
+     * @param locale        - needed language code.
      * @return {@link HabitStatusCalendarDto}.
      */
-    @ApiOperation(value = "Enroll by habit id that is assigned for current user.")
+    @ApiOperation(value = "Enroll habit assign by habitAssignId for current user.")
     @ApiResponses(value = {
         @ApiResponse(code = 200, message = HttpStatuses.OK, response = HabitAssignDto.class),
         @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
         @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
         @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND)
```

Comment:
What response will we have if HabitAssign with `habitAssignId` does not belong to the current user?